js/packages/web/src/components/AuctionNumbers/index.tsx

```diff
@@ -98,56 +104,62 @@ const Countdown = ({ state }: { state?: CountdownState }) => {
           >
             Time left
           </div>
-          {state && (isEnded(state) ? (
-            <Row style={{ width: '100%' }}>
-              <div className="cd-number">This auction has ended</div>
-            </Row>
-          ) : (
-            <Row style={{ width: '100%', flexWrap: 'nowrap' }}>
-              {state && state.days > 0 && (
+          {state &&
```

Comment:
Just my formatter formatting things better than your formatter

Reply:
commit the config so its the same....

Reply:
yeah we probably should